scenes/levels.gd

```diff
@@ -3,14 +3,14 @@ extends Node
 var chapters
 
 func _ready():
-	var lang = game.lang
+	var lang = game.os_lang
```

Comment:
just wondering: what is the variable `lang` used for?

Reply:
sorry ... my mistake!!!

Reply:
excuse me  i don't have sync last commit in pull request! Now all is sync.

Reply:
excuse me  i don't have sync last commit in pull request! Now all is sync.